solidity/ecdsa/contracts/test/upgrades/WalletRegistryV2Invalid.sol

```diff
@@ -0,0 +1,1189 @@
+// This is a contract used to test WalletRegistry upgradeability. It is a copy
```

Comment:
Do you have an idea of a better name for this file? `WalletRegistryV2MissingSlot.sol` is also *Invalid* and it would be great to be explicit about what exactly is *invalid* in this file.

Some ideas:
- `WalletRegistryV2TooEarlySlot.sol`
- `WalletRegistryV2MisplacedNewSlot.sol`

Reply:
6212e48870a16033e4917476a03dda1bb25afd19